tests/idris2/interactive017/expected

```diff
@@ -1,4 +1,4 @@
 1/1: Building RLE (RLE.idr)
 RLE> uncompress Empty = Val []
-uncompress (Run n x rest) = let Val ys = uncompress rest in Val (rep n x ++ ys)
+uncompress (Run n x rest) = let Val ys = uncompress rest in Val (rep n x ++ _)
```

Comment:
Not quite sure about this one. From what I can tell the term coming from the expression search contains an underscore as an argument to the case function, so this is the correct unelaboration, and the fact that the ys was correct there is just a coincidence. (But I have no idea how the expression search works, so I might be wrong.)